Pack-Smart/pack-smart-fe
# Feature/thumbnail me if you can

## What's this PR do?
- GET for getSinglePackingList from ListThumbnail which also uses `history.push` to route the user to `/packing-list` and display that specific list. This also updates the redux store with that packing list from BE
- DELETE for deletePackingList from ListThumbnail and re-rendered page to display without the deleted list.
- Added `Female Typical` and `Male Typical` to gender answer buttons and fixed styling

## Where should the reviewer start?
- apiCalls.js
- ListThumbnails.js
- SavedView.js
- MultipleChoice.js and MultipleChoice.scss for changing gender buttons

## How should this be manually tested?
- Complete a quiz
- Save the packing list
- saved-packing-lists should display all saved lists
- Click a trip card
    * Redux store should be updated with the list for this trip 
    * packing-list should display the packing list that was clicked
- Click Saved Lists button in header
- click `X` and page should re-render without that trip card being displayed
   

## What are the relevant tickets?
- Closes #66 
- Closes #59 
- Closes #58 

## Questions:
- Do we want to reverse the order of the trip cards in SavedView so that the newest created card is first?

## Did you update the README?
- No
